ansible/roles/python-devel/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# tasks file for roles/python-devel
+
+- name: install python-devel
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - "python-devel"
```

Comment:
This can be done in a separate PR. Still trying to come up with a workflow to test these PRs using a Docker with systemctl. Can be discussed at 11am.